pkgs/applications/misc/upwork/default.nix

```diff
@@ -39,13 +39,22 @@ stdenv.mkDerivation rec {
 
   installPhase = ''
     runHook preInstall
+
     mv usr $out
     mv opt $out
+
+    # Now it requires these versions of libssl and libcrypto. Fix based on Spotify pkg.
+    # https://github.com/NixOS/nixpkgs/blob/efea022d6fe0da84aa6613d4ddeafb80de713457/pkgs/applications/audio/spotify/default.nix#L129
+    mkdir -p $out/lib/upwork
+    ln -s ${openssl.out}/lib/libssl.so $out/lib/upwork/libssl.so.1.0.0
+    ln -s ${openssl.out}/lib/libcrypto.so $out/lib/upwork/libcrypto.so.1.0.0
```

Comment:
```suggestion
    ln -s ${lib.getLib openssl}/lib/libssl.so $out/lib/upwork/libssl.so.1.0.0
    ln -s ${lib.getLib openssl}/lib/libcrypto.so $out/lib/upwork/libcrypto.so.1.0.0
```

Reply:
FYI, the result is exactly the same (`openssl` doesn't seem to have a `lib` output). But in the future if someone implements a `lib` output, this will make it future proof.

`getLib` will try `lib` first, than `out` and finally the default output.

Reply:
Understood